From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-memory_hotplug-refactor-hotadd_init_pgdat-and-try_online_node-checkpatch-fixes

WARNING: 'succesfully' may be misspelled - perhaps 'successfully'?
#6: 
whether the node was succesfully allocated.
                     ^^^^^^^^^^^

WARNING: 'succesfully' may be misspelled - perhaps 'successfully'?
#64: FILE: mm/memory_hotplug.c:1188:
+ * 0 -> Either the node was already online, or we succesfully registered a new
                                                   ^^^^^^^^^^^

WARNING: Avoid crashing the kernel - try using WARN_ON & recovery code rather than BUG() or BUG_ON()
#97: FILE: mm/memory_hotplug.c:1206:
+	BUG_ON(ret);

total: 0 errors, 3 warnings, 62 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-memory_hotplug-refactor-hotadd_init_pgdat-and-try_online_node.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memory_hotplug.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/memory_hotplug.c~mm-memory_hotplug-refactor-hotadd_init_pgdat-and-try_online_node-checkpatch-fixes
+++ a/mm/memory_hotplug.c
@@ -1185,7 +1185,7 @@ static void hotadd_init_pgdat(int nid)
  *
  * Returns:
  * 1 -> The node has been initialized.
- * 0 -> Either the node was already online, or we succesfully registered a new
+ * 0 -> Either the node was already online, or we successfully registered a new
  *      one.
  * -errno -> register_one_node() failed.
  */
_
